releases/release-4.0.15.md

```diff
@@ -0,0 +1,125 @@
+---
+title: TiDB 4.0.15 Release Notes
+category: Releases
+---
+
+# TiDB 4.0.15 Release Notes
+
+Release Date: September 27, 2021
+
+TiDB version: 4.0.15
+
+## Improvements
+
++ TiDB
+
+    - Trigger auto-analyze based on histogram row count [#26706](https://github.com/pingcap/tidb/pull/26706)
+
++ TiKV
```

Comment:
```suggestion
  # TiKV

    ## Improvements
        - separate read write ready to reduce read latency [#10619](https://github.com/tikv/tikv/pull/10619)
        - TiKV coprocessor slow log will only consider time spent on processing the request.
        - Drop log instead of blocking threads when slogger thread is overloaded and queue is filled up. [#10863](https://github.com/tikv/tikv/pull/10863)
        - Support changing CDC configs dynamically [#10684](https://github.com/tikv/tikv/pull/10684)
        - Reduce resolved ts message size to save network bandwidth. [#10677](https://github.com/tikv/tikv/pull/10677)
       
    ## Bug Fixes
        - Fix the issue that br reports file already exists error when TDE enabled during restoration. [#10917](https://github.com/tikv/tikv/pull/10917)
        - RaftStore Snapshot GC fix: fix the issue that snapshot GC missed GC snapshot files when there's one snapshot file failed to be GC-ed. [#10871](https://github.com/tikv/tikv/pull/10871)
        - Fix delete stale region too frequently. [#10781](https://github.com/tikv/tikv/pull/10781)
        - fix frequently reconnecting pd client [#9818](https://github.com/tikv/tikv/pull/9818)
        - Check stale file information from encryption file dict [#10598](https://github.com/tikv/tikv/pull/10598)
```